.github/workflows/development.yml

```
@@ -0,0 +1,49 @@
+name: Development
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: ${{matrix.ruby}} on ${{matrix.os}}
+    runs-on: ${{matrix.os}}-latest
+    continue-on-error: ${{matrix.experimental}}
+    
+    strategy:
+      matrix:
+        os:
+          - ubuntu
+          - macos
+        
+        ruby:
+          - "2.6"
+          - "2.7"
+          - "3.0"
+        
+        experimental: [false]
+        env: [""]
+        
+        include:
+          - os: ubuntu
+            ruby: truffleruby
+            experimental: true
+          - os: ubuntu
+            ruby: jruby
+            experimental: true
+          - os: ubuntu
+            ruby: head
+            experimental: true
+    
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Install dependencies
+      run: apt-get install curl
```

Don't it need sudo? Well, you are making this, great, I'll stop making the GitHub Actions PR I was fiddling with.

---

```suggestion
      run: sudo apt-get install libcurl-dev
```

Do we get headers with that other package?